src/include/migraphx/op/broadcast.hpp

```diff
@@ -35,10 +35,16 @@ struct broadcast
     std::string name() const { return "broadcast"; }
     shape compute_shape(std::vector<shape> inputs) const
     {
-        auto t     = inputs.at(0).type();
         auto input = inputs.at(0);
+        auto t     = input.type();
 
         std::vector<size_t> bcast_strides(broadcast_lens.size(), 0);
+        // the broacast op is deprecated now, so not handling the negative
```

Comment:
What do you mean the broadcast op is deprecated? We should normalize the `axis`.

Reply:
This operator does not existed anymore, which mean no new model will use it and existing models donot have such a problem. Anyway, I can change it.

Reply:
After checking our code, we do not parse the broadcast code at all, which means the only scenario we have the broadcast operator is we explicitly add it when necessary. In addition, there is no specification about the range of the axis attribute. And the way of converting negative value to positive is different other operators.

Reply:
Its deprecated in onnx, but it doesn't mean its deprecated in migraphx. Did this operator support negative axis before?

Reply:
No, we do not have negative axis. Actually, the axis is declared as uint64_t.